spec/services/export/report_spec.rb

```diff
@@ -325,7 +337,7 @@
 
         it "returns the rows with the correct tags" do
           first_row = subject.rows.first.to_a
-          tags_index = 55
+          tags_index = 56
```

Comment:
Could you apply the awesome refactoring to the Tags column as well?

Reply:
Yep.